db/migrate/20231010180442_create_mentee_questionnaires.rb

```diff
@@ -0,0 +1,25 @@
+class CreateMenteeQuestionnaires < ActiveRecord::Migration[7.0]
+  def change
+    create_table "mentee_questionnaires", id: :binary, force: :cascade do |t|
+      t.binary "respondent_id", limit: 16, null: false, foreign_key: true
+      t.string :name, null: false
+      t.string :country, null: false
+      t.string :city, null: false
+      t.string :work_url, null: false
+      t.boolean :writing_ruby, null: false
+      t.string :starting_point, null: false
```

Comment:
Not totally sure what this question _is_

Reply:
Starting point is for: 

Where'd you get your start? Which bootcamp? Which university? Self-taught?

Reply:
Updated the model attributes table in the PR description to precisely match with the current form questions

Reply:
Maybe:

```suggestion
      t.string :where_started_coding, null: false
```